pymc3/distributions/continuous.py

```diff
@@ -2478,7 +2507,17 @@ def logcdf(self, value):
         """
         alpha = self.alpha
         beta = self.beta
-        return bound(tt.log(tt.gammainc(alpha, beta * value)), value >= 0, alpha > 0, beta > 0)
+        # To avoid gammainc C-assertion when given invalid values (#4340)
```

Comment:
What does "gammainc" mean exactly?

Reply:
the regularized lower gamma function (also known as Gamma P)

Reply:
I added a more descriptive comment

Reply:
Thanks @domenzain and @ricardoV94 -- 2021 is starting off pretty well: I already learned something 😉